plugins/win-capture/data/locale/en-US.ini

```
@@ -38,6 +38,9 @@ GameCapture.Rgb10a2Space="RGB10A2 Color Space"
 GameCapture.Rgb10a2Space.Srgb="sRGB"
 GameCapture.Rgb10a2Space.2100PQ="Rec. 2100 (PQ)"
 Mode="Mode"
+CaptureAudio="Capture Audio"
+CaptureAudio.TT="When enabled an \"Application Audio Capture\" source will be created that automatically is updated to the currently captured window/application."
```

This wording tripped me up, took me 4 attempts to read it correctly & understand what it was trying to convey.
I'd also specify that the "Desktop Audio" source may conflict.

Something like:
```suggestion
CaptureAudio.TT="When enabled, creates an \"Application Audio Capture\" source that automatically updates to the currently captured window/application. Note that if Desktop Audio is configured, this could result in doubled audio."
```